src/components/Calculator.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import './ButtonPanel.css';
+import './Calculator.css';
+
+class Calculator extends React.Component {
+  render() {
+    return (
+      <div className='component-app'>
+        {/* display zone */}
+        <div style={{ width: 300 }}>
+          <h3 style={{ paddingLeft: 15 }}>000</h3>
+        </div>
+        {/* buttons and signs */}
+        <ButtonPanel />
+      </div>
+    );
+  }
```

Comment:
- [ ] Good job so far 👏 In addition to what you have done, I would suggest that you improve your design in order to match the design of the calculator given. Please take a look at the comparison below 👇 
**Given design**
![image](https://user-images.githubusercontent.com/90222110/213871349-5d02f552-6d3d-4c2d-955c-177c664ed6ea.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/213871370-c0a8ba13-8b3a-4fc4-9e8b-106fecb6044d.png)

As you can see there are places that you still have to work on so kindly improve that too 😃 👍